src/guides/v2.4/install-gde/prereq/os-migration.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Elasticsearch
+functional_areas:
+  - Configuration
+  - Search
+  - System
+  - Setup
+---
+
+{% include config/es-version-24.md %}
+
+## About OpenSearch
+OpenSearch is an open-source fork of Elasticsearch 7.10.2 following Elasticsearch's licensing change.
+
+OpenSearch support has been added to Magento 2.4.4. Elasticsearch remains supported, although it is no longer supported
+for Magento Cloud 2.4.4.
+
+## Migrating to OpenSearch
+The steps to migrate to OpenSearch are simple and largely follow the steps for ElasticSearch configuration.
+1. (optional) Uninstall Elasticsearch. Customers will likely put the site in [Maintenance Mode] if doing this.
+
+2. [Install OpenSearch] and follow the recommended Magento [Elasticsearch system configuration guide].
+
+3.After installing OpenSearch, Customers should follow the guide [Configure Magento to use Elasticsearch].
```

Comment:
Space after the number and OLs can all start with 1. The renderer does the numbering.

Reply:
fixed